etcd3/client.py

```diff
@@ -86,34 +74,94 @@ def __init__(self, alarm_type, member_id):
         self.member_id = member_id
 
 
+class Node(object):
+    """Represents an etcd cluster member."""
+
+    creds = None
+    time_retry = 300.0
```

Comment:
Agreed, it can be configured simply by setting

etcd3.Node.time_retry = X

Do you have something else in mind?